src/main/kotlin/org/rust/ide/refactoring/RsRefactoringSupportProvider.kt

```diff
@@ -12,9 +12,14 @@ import org.rust.ide.refactoring.extractFunction.RsExtractFunctionHandler
 import org.rust.ide.refactoring.introduceParameter.RsIntroduceParameterHandler
 import org.rust.ide.refactoring.introduceVariable.RsIntroduceVariableHandler
 import org.rust.lang.core.psi.RsPatBinding
+import org.rust.lang.core.psi.ext.RsReferenceElement
 
 class RsRefactoringSupportProvider : RefactoringSupportProvider() {
-    override fun isInplaceRenameAvailable(element: PsiElement, context: PsiElement?): Boolean = element is RsPatBinding
+    override fun isInplaceRenameAvailable(element: PsiElement, context: PsiElement?): Boolean =
+        element is RsPatBinding
+
+    override fun isMemberInplaceRenameAvailable(element: PsiElement, context: PsiElement?): Boolean =
+        element is RsReferenceElement
```

Comment:
In what cases does it should work?
Looks like element is always not `RsReferenceElement` even we call rename on reference element

Reply:
That's a really good point!

Reply:
Fixed.